openlibrary/templates/books/add.html

```diff
@@ -20,7 +20,7 @@ <h1>$_("Add a book to Open Library")</h1>
             <div class="label"><label for="title">$_("Title")</label></div>
             <div class="input">
                 $if work:
-                    <input type="text" id="title" disabled="disabled" value="$work.title"/>
+                    <input type="text" id="title" value="$work.title"/>
```

Comment:
@shucon Removing the disabled on the input field is just one part of the solution. 
You would need to pass the new title to the book creating process too. 
So let us consider this case:
When I change the title as shown below: 
http://localhost:8080/books/add?work=/works/OL54120W
![Screenshot from 2020-07-20 11-49-55](https://user-images.githubusercontent.com/31198893/87905914-39f16480-ca7f-11ea-8eb4-c4785e0d6fcb.png)
Then after clicking on Add book and scrolling to the title of the book in the edit form 
http://localhost:8080/books/OL12M/The_wit_wisdom_of_Mark_Twain/edit?mode=add-book#Title here:
![Screenshot from 2020-07-20 11-52-27](https://user-images.githubusercontent.com/31198893/87906047-850b7780-ca7f-11ea-9983-561743b57b71.png)
As you can see the Edition title is still unchanged.
Also to confirm this in the books table The edition has been created with the same title:
![Screenshot from 2020-07-20 11-53-29](https://user-images.githubusercontent.com/31198893/87906157-b3895280-ca7f-11ea-845d-b1284c71c1c5.png)